test/dsl/custom_functions.jl

```diff
@@ -7,6 +6,9 @@ t = default_t()
 using StableRNGs
 rng = StableRNG(12345)
 
+# Fetch test functions.
+include("../test_functions.jl")
```

Comment:
There was no way to declare the helper function in the runtests.jl file, so I put them in a separate script.